torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -257,7 +258,7 @@ bool isWeight(Value* v) {
       v,
       AtenFuncArgs(
           {{"conv1d", 1}, {"conv2d", 1}, {"conv3d", 1}, {"linear", 1}}),
-      CallFuncArgs({{"linear", 2}}));
+      CallFuncArgs({{"linear", 2}, {"embedding_bag", 2}}));
```

Comment:
I feel we should add some docs for these configurations to make it more clear

Reply:
Do you mean adding the op signature here for every op?

Reply:
yeah